backend/modules/simulation/handlers/chat_handler.py

```
@@ -376,19 +504,28 @@ async def handle_stream_message(
+                            if not persona_name or persona_name == "ChatOrchestrator":
+                                persona_name = "ChatOrchestrator"
+                                persona_id = None
+                            
+                            logger.error(
+                                f"Error streaming persona response for persona {persona_id_str} "
+                                f"(persona_name={persona_name}, persona_id={persona_id}): {error_msg}",
+                                exc_info=True
+                            )
                             if _is_dev:
                                 traceback.print_exc()
                             ai_response = f"I'm sorry, I'm having trouble processing that right now. Please try again or ask the orchestrator for help."
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)